scheduler/drivers/scheduler/driver.go

```diff
@@ -21,44 +23,66 @@ type Config struct {
 	Name          string
 	L1Client      *ethclient.Client
 	SequencerAddr common.Address // sequencer contract address
+	TSHAddr       common.Address // sequencer staking address
 }
 
 // Driver implements the service.Driver interface.
 var _ service.Driver = &Driver{}
 
 type Driver struct {
-	cfg               Config
-	sequencerContract *scc.StateCommitmentChain
-	seqz              *SequencerSet
-	block             uint64     // current block height
-	producer          *Sequencer // currenct producer
-	metrics           *metrics.Base
+	cfg         Config
+	tgmContract *tgm.TssGroupManager
+	tshContract *tsh.TssStakingSlashing
+	seqz        *SequencerSet
+	block       uint64     // current block height
+	producer    *Sequencer // currenct producer
+	metrics     *metrics.Base
 }
 
 func NewDriver(cfg Config) (*Driver, error) {
-	sequencerContract, err := scc.NewStateCommitmentChain(
+	tgmContract, err := tgm.NewTssGroupManager(
 		cfg.SequencerAddr, cfg.L1Client,
 	)
 	if err != nil {
 		return nil, err
 	}
 
-	// TODO: get sequencers from Sequencer contract
-	// d.sequencerContract.getSequencers()
-	sequencers := []common.Address{}
+	tshContract, err := tsh.NewTssStakingSlashing(
+		cfg.TSHAddr, cfg.L1Client,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	// get sequencers from Sequencer contract
+	pubKeys, err := tgmContract.GetTssGroupMembers(nil)
+	if err != nil {
+		return nil, err
+	}
 	var seqz []*Sequencer
-	for _, sequencer := range sequencers {
+	for _, v := range pubKeys {
+		ecdsaPubKey, err := crypto.UnmarshalPubkey(v)
+		if err != nil {
+			return nil, err
+		}
+		addr := crypto.PubkeyToAddress(*ecdsaPubKey)
+		deposit, err := tshContract.GetDeposits(nil, common.BytesToAddress(addr.Bytes()))
```

Comment:
To be determined : Contract provide the method for scheduler query a batch of deposit